packages/cli/src/commands/account/new.ts

```diff
@@ -136,7 +145,7 @@ export default class NewAccount extends BaseCommand {
     if (derivationPath === CELO_DERIVATION_PATH_BASE) {
       this.log(
         chalk.magenta(
-          `\nUsing celo-legacy path (${CELO_DERIVATION_PATH_BASE}) for derivation. This will be switched to eth derivation path (${ETHEREUM_DERIVATION_PATH}) next major version.\n`
+          `\nUsing celoLegacy path (${CELO_DERIVATION_PATH_BASE}) for derivation. This will be switched to eth derivation path (${ETHEREUM_DERIVATION_PATH}) next major version.\n`
```

Comment:
Hmmm I know this isn't from this PR but now the phrasing leads to believe that the current derivation path will be switched to eth path instead. But what we want to convey is that the default will change, but users can keep using the ones in their configs. No sure how to phrase it tho

Reply:
i said defaulted to instead of switched